package.json

```diff
@@ -60,12 +60,12 @@
     "style-loader": "0.21.0",
     "stylus": "^0.54.5",
     "stylus-loader": "3.0.2",
-    "uglify-js": "2.8.29",
-    "uglifyjs-webpack-plugin": "1.2.5",
     "url-loader": "1.0.1",
-    "webpack": "3.11.0",
+    "webpack": "4.8.3",
     "webpack-bundle-analyzer": "2.11.1",
-    "webpack-dev-server": "2.11.1"
+    "webpack-cli": "^2.1.3",
```

Comment:
Please use exact versions of packages (e. g. `2.1.3` instead of `^2.1.3`) because it reduces “works on my computer” kind of errors — everybody will use the same version of package.